salesagility/SuiteCRM
# Rebrand the Search Filter in Modules

SCRM-205 - Replace occurrences of "Search" with "Filter" where the user could see them, especially in studio.
SCRM-204 - Remove "Search" from listview header.
Remove duplicated deceleration of 'LBL_ADVANCED_SEARCH' in en_us.lang.php

## Description

Changing labels for "Search" to "Filter".
Removing "Search" from List View headers.
Remove duplicate deceleration of 'LBL_ADVANCED_SEARCH' (Issue #2979)




## Motivation and Context

Occurrences of "Search" had to be replaced with "Filter" for the next release.
"Search"  had to be removed from the List View headers.

## How To Test This

Browse to locations "Search" was located previously such as Studio.
View any list view page and see if "Search" is included in the header.

## Types of changes


### Final checklist
